spec/models/profile.rb

```diff
@@ -6,7 +6,7 @@ class Profile
   field :age
   field :hobbies, type: Array, default: []
 
-  embedded_in :user, inverse_of: :profile
+  embedded_in :user, inverse_of: :profile, touch: false
```

Comment:
Embedded associations now have the touch option defaulted to `true`: https://www.mongodb.com/docs/mongoid/master/release-notes/mongoid-9.0/#embedded_in-associations-now-default-to-touch--true